clients/windows/lockbook/SignUp.xaml

```diff
@@ -128,7 +128,12 @@
                     <TextBox
                         Width="300"
                         x:Name="usernameTextBox"
-                        Header="Select a Username" />
+                        Header="Username" />
+                    <TextBox
```

Comment:
Not on board with all users seeing this, it should be behind some debug config or something